frappe/public/js/frappe/ui/toolbar/about.js

```diff
@@ -10,20 +10,20 @@ frappe.ui.misc.about = function () {
 					__("Open Source Applications for the Web") +
 					"</p>  \
 		<p><i class='fa fa-globe fa-fw'></i>\
-			Website: <a href='https://frappeframework.com' target='_blank'>https://frappeframework.com</a></p>\
+			__("Website"): <a href='https://frappeframework.com' target='_blank'>https://frappeframework.com</a></p>\
```

Comment:
This is adding JS function inside HTML. It won't work like this 😅 

Use template strings `<html>... ${__("text")} `

Reply:
oh ok 😅